### ISSUE TITLE
(some?) metapackages cannot be uninstalled. 

### ISSUE THREAD

**Issue by mingwandroid** [conda/conda#4182]
On the latest 4.2.x branch, on Windows, trying to upgrade my r-essentials to 1.5.1 I'm getting the following:

```
    Traceback (most recent call last):
      File "C:\Users\builder\mc-x64-3.5\lib\site-packages\conda\exceptions.py", line 479, in conda_exception_handler
        return_value = func(*args, **kwargs)
      File "C:\Users\builder\mc-x64-3.5\lib\site-packages\conda\cli\main.py", line 145, in _main
        exit_code = args.func(args, p)
      File "C:\Users\builder\mc-x64-3.5\lib\site-packages\conda\cli\main_install.py", line 80, in execute
        install(args, parser, 'install')
      File "C:\Users\builder\mc-x64-3.5\lib\site-packages\conda\cli\install.py", line 405, in install
        execute_actions(actions, index, verbose=not context.quiet)
      File "C:\Users\builder\mc-x64-3.5\lib\site-packages\conda\plan.py", line 643, in execute_actions
        inst.execute_instructions(plan, index, verbose)
      File "C:\Users\builder\mc-x64-3.5\lib\site-packages\conda\instructions.py", line 135, in execute_instructions
        cmd(state, arg)
      File "C:\Users\builder\mc-x64-3.5\lib\site-packages\conda\instructions.py", line 83, in UNLINK_CMD
        unlink(state['prefix'], arg)
      File "C:\Users\builder\mc-x64-3.5\lib\site-packages\conda\install.py", line 1040, in unlink
        mk_menus(prefix, meta['files'], remove=True)
    KeyError: 'files'
```

I added a debug log to see what meta contained and it is:
```
DEBUG conda.install:unlink(1038): meta {'build': 'r3.3.1_0', 'build_number': 0, 'subdir': 'win-64', 'priority': 0, 'version': '3.3.1', 'name': 'r', 'depends': ['r-base 3.3.1', 'r-recommended 3.3.1'], 'md5': '571a84d2ba62cba6dcddb7a9cf10a0a4', 'size': 589, 'channel': 'https://conda.anaconda.org/r', 'url': 'https://conda.anaconda.org/r/win-64/r-3.3.1-r3.3.1_0.tar.bz2', 'platform': 'win32', 'fn': 'r-3.3.1-r3.3.1_0.tar.bz2', 'arch': 'x86_64', 'link': {'type': 'hard-link', 'source': 'C:\\Users\\builder\\mc-x64-3.5\\pkgs\\r-3.3.1-r3.3.1_0'}, 'schannel': 'r'}
```

Here, `r` is a metapackage. Has the handling of meta['files'] changed from being an empty list when not present to being missing entirely?

**Comment by y2kbugger** [comment]
I have reproduced this with a different package, though I didn't figure out which it was—I was `conda install`ing a tar to move an existing environment to a different PC offline. Upon receiving the above error I went through the tar and manually removed all packages which were already installed in the target env (a clone of the default miniconda 4.2.12.0 root env). After removing the packages everything installed fine.

_Note: The second time I did this I installed directly to the root env to hedge my bet of just getting it working. I am pretty sure that it was removing the packages from the tar that actually fixed the problem_

_Note 2: When purging the tar I removed the conda package even though the tar had a newer version. (4.2.13 vs 4.2.12)

This was on 64bit Windows 7


I don't see an explicit fix for this in the change log, but next week I will test with new version of conda and give feedback.
